js/cssmenu.js

```diff
@@ -0,0 +1,25 @@
+( function( $ ) {
+$( document ).ready(function() {
+var menu_ul = $('#cssmenu > ul > li > ul');
+menu_ul.hide();
+
+$('#cssmenu > ul > li > a').click(function() {
+  $('#cssmenu li').removeClass('active');
+  $(this).closest('li').addClass('active');
+  var checkElement = $(this).next();
+  if((checkElement.is('ul')) && (checkElement.is(':visible'))) {
+    $(this).closest('li').removeClass('active');
+    checkElement.slideUp('normal');
```

Comment:
Feels a little sluggish at the moment, can you try `'fast'` instead of `'normal'`? (`slideDown` 3 lines below too)